src/overlays/actors/ovl_Obj_Kendo_Kanban/z_obj_kendo_kanban.c

```diff
@@ -155,13 +170,14 @@ static DamageTable sDamageTable = {
 
 static CollisionCheckInfoInit2 sColChkInfoInit = { 8, 0, 0, 0, MASS_HEAVY };
 
-Vec3f D_80B6681C = { 0.0f, 0.0f, 0.0f };
-Vec3f D_80B66828 = { 1.0f, 0.0f, 0.0f };
+static Vec3f sNullVec3F = { 0.0f, 0.0f, 0.0f };
```

Comment:
```suggestion
static Vec3f sZeroVec = { 0.0f, 0.0f, 0.0f };
```
This is usually what we call vectors like this.